vehicles.sql

```diff
@@ -15,12 +15,8 @@ CREATE TABLE IF NOT EXISTS `player_vehicles` (
   `depotprice` int(11) NOT NULL DEFAULT 0,
   `drivingdistance` int(50) DEFAULT NULL,
   `status` text DEFAULT NULL,
-  `balance` int(11) NOT NULL DEFAULT 0,
-  `paymentamount` int(11) NOT NULL DEFAULT 0,
-  `paymentsleft` int(11) NOT NULL DEFAULT 0,
-  `financetime` int(11) NOT NULL DEFAULT 0,
   PRIMARY KEY (`id`),
-  KEY `plate` (`plate`),
-  KEY `citizenid` (`citizenid`),
-  KEY `license` (`license`)
-) ENGINE=InnoDB AUTO_INCREMENT=1;
+  UNIQUE KEY `plate` (`plate`),
```

Comment:
Should we enforce this? Is it enforced in-game that no two vehicles can have the same plate by GTA itself?

Reply:
In game no, but as we use the plate as a (supposedly unique) key it needs to be unique.

Whenever we refactor the db and use a proper unique identifier we can have it be not unique tho

Reply:
If this isn't enforced currently, I don't think we should add the constraint since we don't want to use the plate as a unique key and for backwards compatibility, it's possible that others are doing the same

Reply:
Plate is used as a pseudo primary key in basically every script, it is clearly assumed that a plate is unique.
Also it just makes sense, license plates ARE unique.